Week1/homework/assignment.json

```diff
@@ -0,0 +1,42 @@
+[
+  {
+    "name": "1. What are the names of the countries with population greater than 8 million?",
+    "query": "SELECT name FROM country WHERE population > 8000000;"
+  },
+  {
+    "name": "2. What are the names of the countries that have “land” in their names? ",
+    "query": "SELECT name FROM country WHERE name LIKE '%land%';"
+  },
+  {
+    "name": "3. What are the names of the cities with population in between 500,000 and 1 million?",
+    "query": "SELECT name FROM city WHERE population BETWEEN 500000 AND 1000000;"
+  },
+  {
+    "name": "4. What are the names of all the countries on the continent \"Europe\"?",
+    "query": "SELECT name FROM country WHERE Continent = 'Europe';"
+  },
+  {
+    "name": "5. List all the countries in the descending order based on their surface areas.",
+    "query": "SELECT name FROM country ORDER BY SurfaceArea DESC;"
+  },
+  {
+    "name": "6. What are the names of all the cities in the Netherlands?",
+    "query": "SELECT city.name AS CityName FROM city INNER JOIN country ON city.CountryCode = country.Code WHERE country.name = 'Netherlands';"
+  },
```

Comment:
It's a good Join , Inner is not necessary keyword as it's inner by default

Reply:
Yes, you are right,..
Thank you very much for review,..